### PR TITLE
feat: turn on arbitrum nova production

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

Turns on arbitrum nova in prod

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

Low

## Testing

Supported:
- Limited assets
- Send/Receive
- Balances
- Transaction history
- Wallets: ShapeShift, Coinbase, Ledger, MetaMask, WalletConnect V2

Unsupported:
- On ramps
- All defi functionality
- NFT view
- Wallets: KeepKey, XDEFI, WalletConnect V1

### Engineering

:point_up: 

### Operations

:point_up: 

## Screenshots (if applicable)

image

image

image

### CODE REVIEW

File: src/lib/swapper/swappers/utils/filterBuyAssetsBySellAssetId/filterBuyAssetsBySellAssetId.ts [hunk 1]
```diff
@@ -13,8 +13,12 @@ const _filterEvmBuyAssetsBySellAssetId = (
   if (!isEvmChainId(sellAsset.chainId)) return []
 
   return assets.filter(buyAsset => {
-    // evm only AND chain id predicate
-    return isEvmChainId(buyAsset.chainId) && chainIdPredicate(buyAsset.chainId, sellAsset.chainId)
+    // evm only AND chain id predicate with no arbitrum nova support for any swappers
```
Comment (gomesalexandre) [inline src/lib/swapper/swappers/utils/filterBuyAssetsBySellAssetId/filterBuyAssetsBySellAssetId.ts:16]: 🧠

File: src/lib/swapper/swappers/utils/filterAssetIdsBySellable/filterAssetIdsBySellable.ts [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 import type { Asset } from '@shapeshiftoss/types'
 
-// we dont perform a lookup to lifi's supported assets because they support far more assets than we do
-// so the overhead in performing the fetch to lifi isnt worth the time
 export function filterEvmAssetIdsBySellable(assets: Asset[]): Asset[] {
   return assets.filter(asset => {
     const { chainId } = asset
 
-    return evmChainIds.includes(chainId as EvmChainId)
+    // evm only with no arbitrum nova support for any swappers
```
Comment (gomesalexandre) [inline src/lib/swapper/swappers/utils/filterAssetIdsBySellable/filterAssetIdsBySellable.ts:9]: This isn't working as it should - Nova is still available as a buy and sell asset

image
image

**Review by gomesalexandre — APPROVED** [review]
gm, stamped pending the swapper issue is tackled, although this is non-blocking

### Tx history

image

### `ethSwitchChain` and add chain to MM

image
image

### Self-send

image
image

### Wallet accounts entries

- Account 0

image

- Account 1 derivation on native

image

File: src/lib/swapper/swappers/utils/filterAssetIdsBySellable/filterAssetIdsBySellable.ts (hunk 2 above)
Comment (kaladinlight) [reply src/lib/swapper/swappers/utils/filterAssetIdsBySellable/filterAssetIdsBySellable.ts:9]: Cache clear fixed as per dms

**Pull request merged**